src/entity_resolver/scripts.py

```diff
@@ -1,5 +1,7 @@
 import argparse
 from functools import wraps
+import json
+
```

Comment:
Move `import json` before the last line: `from functools import wraps` and also the newly added empty line. According to pep8 standard, there should be exactly 2 empty lines between import statements and the first line of code.